package.json

```diff
@@ -28,12 +28,13 @@
   "author": "Antonio Salazar Cardozo <antonio@thesis.co>",
   "license": "MIT",
   "dependencies": {
-    "@keep-network/tbtc": ">0.15.0-pre <0.15.0-rc",
     "@keep-network/keep-ecdsa": ">0.15.0-pre <0.15.0-rc",
+    "@keep-network/tbtc": ">0.15.0-pre <0.15.0-rc",
     "bcoin": "git+https://github.com/bcoin-org/bcoin.git#8851582",
     "bcrypto": "^4.1.0",
     "bufio": "^1.0.6",
-    "electrum-client-js": "git+https://github.com/keep-network/electrum-client-js.git#v0.1.0"
+    "electrum-client-js": "git+https://github.com/keep-network/electrum-client-js.git#v0.1.0",
+    "web3-utils": "^1.2.8"
```

Comment:
I think we actually need to move web3 and its provider packages to `peerDependencies`. The binaries that come with tbtc.js require all of those to work as well.